src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -228,28 +228,28 @@ export class FocusMonitor implements OnDestroy {
    * @param origin Focus origin.
    * @param options Options that can be used to configure the focus behavior.
    */
-  focusVia(element: HTMLElement, origin: FocusOrigin, options?: FocusOptions): void;
+  focusVia(element: HTMLElement, origin: NonNullable<FocusOrigin>, options?: FocusOptions): void;
 
   /**
    * Focuses the element via the specified focus origin.
    * @param element Element to focus.
    * @param origin Focus origin.
    * @param options Options that can be used to configure the focus behavior.
    */
-  focusVia(element: ElementRef<HTMLElement>, origin: FocusOrigin, options?: FocusOptions): void;
+  focusVia(element: ElementRef<HTMLElement>,
+           origin: NonNullable<FocusOrigin>,
+           options?: FocusOptions): void;
 
   focusVia(element: HTMLElement | ElementRef<HTMLElement>,
-          origin: FocusOrigin,
-          options?: FocusOptions): void {
+           origin: NonNullable<FocusOrigin>,
+           options?: FocusOptions): void {
 
     const nativeElement = coerceElement(element);
-
     this._setOriginForCurrentEventQueue(origin);
 
-    // `focus` isn't available on the server
     if (typeof nativeElement.focus === 'function') {
-      // Cast the element to `any`, because the TS typings don't have the `options` parameter yet.
-      (nativeElement as any).focus(options);
+      // `focus` isn't available on the server.
```

Comment:
nit: I think it makes sense to leave this comment where it was, since it's explaining the `if`, not the call to `focus`